src/client.py

```diff
@@ -0,0 +1,30 @@
+"""This file is the client module of the CF 401 Python HTTP-Server assignment."""
+
+import sys
+import socket
+
+
+def client(message):
+    """Send given message to server and recover any reply."""
+    infos = socket.getaddrinfo('127.0.0.1', 5002)
+    stream_info = [i for i in infos if i[1] == socket.SOCK_STREAM][0]
+
+    clnt = socket.socket(*stream_info[:3])
+    clnt.connect(stream_info[-1])
+
+    print("Sending: ", message)
+    clnt.sendall(message)
```

Comment:
Message should be encoded into bytestring, so a message can still be sent via client from the command line.